extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -106,6 +111,25 @@ export class AmpAd extends AMP.BaseElement {
       });
     });
   }
+
+  /** @param {string} type */
+  maybeSelectIntoDblckDelayedFetchWhitelistDeprecationExperiment(type) {
+    switch (type) {
```

Comment:
if (isExperimentOn(this.win, 'dcdf-whitelist-deprecation') && !!['ix', 'imonomy', 'medianet', 'navegg', 'openx', 'pulsepoint', 'rubicon', 'yieldbot', 'criteo', 'doubleclick].find(item => item == type)) {
   addExperimentIdToElement(this.element, Math.random() < 0.5 ? '21061861' : '21061862');
}